Add support for python 3.13

Python 3.13 has been released last year and already received first patch update (sometimes considered as a maturity marker). It doesn't seem to have many breaking changes that would affect `clickhouse-driver`.

Could you generate wheels for this platform as well? It would be nice to build wheels for `clickhouse-cityhash` as well as it doesn't compile with Python 3.13. I see there is a [PR](https://github.com/xzkostyan/clickhouse-cityhash/pull/5) addressing that.

```
Building wheels for collected packages: clickhouse-cityhash
  Building wheel for clickhouse-cityhash (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for clickhouse-cityhash (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [114 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build/lib.linux-x86_64-cpython-313/clickhouse_cityhash
      copying clickhouse_cityhash/__init__.py -> build/lib.linux-x86_64-cpython-313/clickhouse_cityhash
      running build_ext
      building 'clickhouse_cityhash.cityhash' extension
      creating build/temp.linux-x86_64-cpython-313/src
      g++ -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O3 -Wall -fPIC -Iinclude -I/home/kammala/data/prog/work/trading-streams/py313/include -I/home/kammala/.asdf/installs/python/3.13.1/include/python3.13 -c src/city.cc -o build/temp.linux-x86_64-cpython-313/src/city.o -O3 -Wno-unused-value -Wno-unused-function
      g++ -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O3 -Wall -fPIC -Iinclude -I/home/kammala/data/prog/work/trading-streams/py313/include -I/home/kammala/.asdf/installs/python/3.13.1/include/python3.13 -c src/cityhash.cpp -o build/temp.linux-x86_64-cpython-313/src/cityhash.o -O3 -Wno-unused-value -Wno-unused-function
      src/cityhash.cpp: In function ‘void __Pyx_ErrRestoreInState(PyThreadState*, PyObject*, PyObject*, PyObject*)’:
      src/cityhash.cpp:3045:24: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_type’
       3045 |     tmp_type = tstate->curexc_type;
            |                        ^~~~~~~~~~~
      src/cityhash.cpp:3046:25: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_value’
       3046 |     tmp_value = tstate->curexc_value;
            |                         ^~~~~~~~~~~~
      src/cityhash.cpp:3047:22: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_traceback’
       3047 |     tmp_tb = tstate->curexc_traceback;
            |                      ^~~~~~~~~~~~~~~~
      src/cityhash.cpp:3048:13: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_type’
       3048 |     tstate->curexc_type = type;
            |             ^~~~~~~~~~~
      src/cityhash.cpp:3049:13: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_value’
       3049 |     tstate->curexc_value = value;
            |             ^~~~~~~~~~~~
      src/cityhash.cpp:3050:13: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_traceback’
       3050 |     tstate->curexc_traceback = tb;
            |             ^~~~~~~~~~~~~~~~
      src/cityhash.cpp: In function ‘void __Pyx_ErrFetchInState(PyThreadState*, PyObject**, PyObject**, PyObject**)’:
      src/cityhash.cpp:3056:21: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_type’
       3056 |     *type = tstate->curexc_type;
            |                     ^~~~~~~~~~~
      src/cityhash.cpp:3057:22: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_value’
       3057 |     *value = tstate->curexc_value;
            |                      ^~~~~~~~~~~~
      src/cityhash.cpp:3058:19: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_traceback’
       3058 |     *tb = tstate->curexc_traceback;
            |                   ^~~~~~~~~~~~~~~~
      src/cityhash.cpp:3059:13: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_type’
       3059 |     tstate->curexc_type = 0;
            |             ^~~~~~~~~~~
      src/cityhash.cpp:3060:13: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_value’
       3060 |     tstate->curexc_value = 0;
            |             ^~~~~~~~~~~~
      src/cityhash.cpp:3061:13: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_traceback’
       3061 |     tstate->curexc_traceback = 0;
            |             ^~~~~~~~~~~~~~~~
      src/cityhash.cpp: In function ‘void __Pyx_Raise(PyObject*, PyObject*, PyObject*, PyObject*)’:
      src/cityhash.cpp:3214:36: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_traceback’
       3214 |         PyObject* tmp_tb = tstate->curexc_traceback;
            |                                    ^~~~~~~~~~~~~~~~
      src/cityhash.cpp:3217:21: error: ‘PyThreadState’ {aka ‘struct _ts’} has no member named ‘curexc_traceback’
       3217 |             tstate->curexc_traceback = tb;
            |                     ^~~~~~~~~~~~~~~~
      src/cityhash.cpp: In function ‘uint64_t __Pyx_PyInt_As_uint64_t(PyObject*)’:
      src/cityhash.cpp:3605:55: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
       3605 |             const digit* digits = ((PyLongObject*)x)->ob_digit;
            |                                                       ^~~~~~~~
      src/cityhash.cpp:3660:55: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
       3660 |             const digit* digits = ((PyLongObject*)x)->ob_digit;
            |                                                       ^~~~~~~~
      src/cityhash.cpp:3746:46: error: too few arguments to function ‘int _PyLong_AsByteArray(PyLongObject*, unsigned char*, size_t, int, int, int)’
       3746 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
            |                           ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
       3747 |                                               bytes, sizeof(val),
            |                                               ~~~~~~~~~~~~~~~~~~~
       3748 |                                               is_little, !is_unsigned);
            |                                               ~~~~~~~~~~~~~~~~~~~~~~~~
      In file included from /home/kammala/.asdf/installs/python/3.13.1/include/python3.13/longobject.h:107,
                       from /home/kammala/.asdf/installs/python/3.13.1/include/python3.13/Python.h:81,
                       from src/cityhash.cpp:4:
      /home/kammala/.asdf/installs/python/3.13.1/include/python3.13/cpython/longobject.h:111:17: note: declared here
        111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
            |                 ^~~~~~~~~~~~~~~~~~~
      src/cityhash.cpp: In function ‘long int __Pyx_PyInt_As_long(PyObject*)’:
      src/cityhash.cpp:3825:55: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
       3825 |             const digit* digits = ((PyLongObject*)x)->ob_digit;
            |                                                       ^~~~~~~~
      src/cityhash.cpp:3880:55: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
       3880 |             const digit* digits = ((PyLongObject*)x)->ob_digit;
            |                                                       ^~~~~~~~
      src/cityhash.cpp:3966:46: error: too few arguments to function ‘int _PyLong_AsByteArray(PyLongObject*, unsigned char*, size_t, int, int, int)’
       3966 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
            |                           ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
       3967 |                                               bytes, sizeof(val),
            |                                               ~~~~~~~~~~~~~~~~~~~
       3968 |                                               is_little, !is_unsigned);
            |                                               ~~~~~~~~~~~~~~~~~~~~~~~~
      /home/kammala/.asdf/installs/python/3.13.1/include/python3.13/cpython/longobject.h:111:17: note: declared here
        111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
            |                 ^~~~~~~~~~~~~~~~~~~
      src/cityhash.cpp: In function ‘int __Pyx_PyInt_As_int(PyObject*)’:
      src/cityhash.cpp:4014:55: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
       4014 |             const digit* digits = ((PyLongObject*)x)->ob_digit;
            |                                                       ^~~~~~~~
      src/cityhash.cpp:4069:55: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
       4069 |             const digit* digits = ((PyLongObject*)x)->ob_digit;
            |                                                       ^~~~~~~~
      src/cityhash.cpp:4155:46: error: too few arguments to function ‘int _PyLong_AsByteArray(PyLongObject*, unsigned char*, size_t, int, int, int)’
       4155 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
            |                           ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
       4156 |                                               bytes, sizeof(val),
            |                                               ~~~~~~~~~~~~~~~~~~~
       4157 |                                               is_little, !is_unsigned);
            |                                               ~~~~~~~~~~~~~~~~~~~~~~~~
      /home/kammala/.asdf/installs/python/3.13.1/include/python3.13/cpython/longobject.h:111:17: note: declared here
        111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
            |                 ^~~~~~~~~~~~~~~~~~~
      src/cityhash.cpp: In function ‘Py_ssize_t __Pyx_PyIndex_AsSsize_t(PyObject*)’:
      src/cityhash.cpp:4447:47: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
       4447 |     const digit* digits = ((PyLongObject*)b)->ob_digit;
            |                                               ^~~~~~~~
      error: command '/usr/bin/g++' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for clickhouse-cityhash
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Native `JSON` support. Solves issue [#473](https://github.com/mymarilyn/clickhouse-driver/issues/473) and [#460](https://github.com/mymarilyn/clickhouse-driver/issues/460). Pull request [#503](https://github.com/mymarilyn/clickhouse-driver/pull/503/) by [khvn26](https://github.com/khvn26), [gsergey418](https://github.com/gsergey418).
+- Minimum `clickhouse-cityhash` version raised to 1.0.2.6: first release that builds on Python 3.13+.
 
 ## [0.2.10] - 2026-10-10
 ### Added

--- a/setup.py
+++ b/setup.py
@@ -130,9 +130,9 @@ setup(
         'lz4': [
             'lz4<=3.0.1; implementation_name=="pypy"',
             'lz4; implementation_name!="pypy"',
-            'clickhouse-cityhash>=1.0.2.1'
+            'clickhouse-cityhash>=1.0.2.6'
         ],
-        'zstd': ['zstd', 'clickhouse-cityhash>=1.0.2.1'],
+        'zstd': ['zstd', 'clickhouse-cityhash>=1.0.2.6'],
         'numpy': ['numpy>=1.12.0', 'pandas>=0.24.0']
     },
     test_suite='pytest'

--- a/testsrequire.py
+++ b/testsrequire.py
@@ -8,7 +8,7 @@ tests_require = [
     'parameterized',
     'freezegun',
     'zstd',
-    'clickhouse-cityhash>=1.0.2.1'
+    'clickhouse-cityhash>=1.0.2.6'
 ]
 
 if sys.implementation.name == 'pypy':

